my-dsl.md

```diff
@@ -1,39 +1,54 @@
 # Language
 
-_What is the name of the language? Link the name to its webpage
-(if appropriate)._
+_What is the name of the language? Link the name to its webpage (if appropriate)._
+
+The DSL that I chose is Regualr Languages or Regex Expressions. The reason why I chose this DSL is because I have worked with them before in a prior internship. [java.util.regex regex link for java](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html)
 
 # Domain
 
 _Describe the language's domain in five words._
 
+External DSL applying pattern recognition 
+
 # Computational model
 
 _We don't yet have a great definition of the term "computational model".
 For now, try to come up with the clearest, most concise explanation of
 what happens when a program in your DSL runs._
 
+Regular Langages analyze and find patterns in strings of code through recursive pattern analysis where the pattern is defined by the user. For instance lets say the user wanted to find a phone number from a list of peoples personal information that may include a bunch of other information that is not impornta to the user. The user could define a regular expression to find phone numbers. This DSL would start by recusivly running through the list and it would itentify anything in the list that matches the pattern of a phone number (+1(123) 456-6789).
+
 # DSL-ness
 
 _Fowler writes about a spectrum of languages, from general-purpose languages to
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum,
 and why?_
 
+Regular Languages fall in the spectrum of more "purely" domain-specific becuase it's language is used as a tool within general-purpose languages such as java and python. Regualr languages abide by all of Fowler's definitions of a DSL and fall into the external DSL catagory. 
+
```

Comment:
I think I agree with you about Fowler's definitions. Though I did have to go look them up to double check that I was remembering - I might have been a bit more specific about how the specific subset of problems that regex is able to solve is why it matches the elements of a DSL, like language nature and and limited expressiveness.